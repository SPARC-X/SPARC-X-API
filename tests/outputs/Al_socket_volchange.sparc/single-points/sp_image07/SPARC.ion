# Ion File Generated by SPARC ASE Calculator
# 
# ASE-SORT:
# 0 1 2 3
# END ASE-SORT

ATOM_TYPE: Al
N_TYPE_ATOM: 4
PSEUDO_POT: 13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
COORD:
0.31307379770930 -0.08628841416725 0.00607806983425
0.07546922833488 3.60405834779348 3.75054402669773
3.74999660335244 -0.32496293980352 3.93862479892939
3.86136971466121 3.63433632847127 -0.03176956464030

