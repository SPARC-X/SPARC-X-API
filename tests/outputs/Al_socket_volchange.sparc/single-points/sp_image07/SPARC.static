***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Al:
      0.0417413754       0.9884953736       0.0008103744
      0.0100621304       0.4805204190       0.5000510018
      0.4999780152       0.9566734739       0.5251273582
      0.5148271238       0.4845573093       0.9957642405
Total free energy (Ha): -9.034213680048802E+00
Atomic forces (Ha/Bohr):
 -1.0180268410E-02  -1.3545411899E-03   3.7172707548E-03
 -8.8847081615E-04  -2.3416680362E-03  -4.5091589004E-04
  7.7536505809E-03   7.7803685287E-03  -8.4374529685E-03
  3.3150886455E-03  -4.0841593026E-03   5.1710981038E-03
Stress (GPa): 
 -1.7027746650E+01  -2.2828391408E-01   5.4052045902E-01 
 -2.2828391408E-01  -1.5863178018E+01   5.1374020884E-01 
  5.4052045902E-01   5.1374020884E-01  -1.6615330738E+01
