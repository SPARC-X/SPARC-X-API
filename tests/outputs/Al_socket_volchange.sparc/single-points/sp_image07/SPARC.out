***************************************************************************
*                       SPARC (version Oct 31, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Thu Jan 18 08:03:31 2024                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 1 1 1 
LATVEC:
7.500322993446770 0.000000000000000 0.000000000000000 
0.000000000000000 7.500322993446770 0.000000000000000 
0.000000000000000 0.000000000000000 7.500322993446770 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.000950043469
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 12
CHEB_DEGREE: 17
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 1
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-03
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-05
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 2.50E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0
MIXING_PARAMETER: 1
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
NUM_CHEFSI: 1
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: SPARC
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
7.500322993446770 0.000000000000000 0.000000000000000 
0.000000000000000 7.500322993446770 0.000000000000000 
0.000000000000000 0.000000000000000 7.500322993446770 
Volume: 4.2192950749E+02 (Bohr^3)
Density: 2.5579190856E-01 (amu/Bohr^3), 2.8663716578E+00 (g/cc)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 2
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 2
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  2
Mesh spacing                       :  0.500022 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  SPARC.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
Atomic mass                        :  26.9815385
Pseudocharge radii of atom type 1  :  7.00 7.00 7.00 (x, y, z dir)
Number of atoms of type 1          :  4
Estimated total memory usage       :  3.01 MB
Estimated memory per processor     :  1.50 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2589546464E+00        1.534E-01        0.021
2            -2.2613160760E+00        2.225E-01        0.007
3            -2.2589296866E+00        1.092E-01        0.006
4            -2.2591118493E+00        1.361E-01        0.007
5            -2.2585842943E+00        3.933E-02        0.007
6            -2.2585935338E+00        4.178E-02        0.007
7            -2.2585565337E+00        6.457E-03        0.006
8            -2.2585549708E+00        2.330E-03        0.006
9            -2.2585534855E+00        1.194E-03        0.007
10           -2.2585534200E+00        9.318E-05        0.006
Total number of SCF: 10    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2585534200E+00 (Ha/atom)
Total free energy                  : -9.0342136800E+00 (Ha)
Band structure energy              :  6.9111882079E-01 (Ha)
Exchange correlation energy        : -4.4225565619E+00 (Ha)
Self and correction energy         : -1.2498741184E+01 (Ha)
-Entropy*kb*T                      : -3.6225459013E-03 (Ha)
Fermi level                        :  1.4538542335E-01 (Ha)
RMS force                          :  8.6734911007E-03 (Ha/Bohr)
Maximum force                      :  1.3850770533E-02 (Ha/Bohr)
Time for force calculation         :  0.014 (sec)
Pressure                           :  1.6502085135E+01 (GPa)
Maximum stress                     :  1.7027746650E+01 (GPa)
Time for stress calculation        :  0.026 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  0.209 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
